### PR TITLE
Add brotli support to static handling

### PR DESCRIPTION
**Pull request by funkybob**
Backport of my patch from master.

### CODE REVIEW

File: core/static.c [hunk 1]
```diff
@@ -3,10 +3,16 @@
 extern struct uwsgi_server uwsgi;
 
 int uwsgi_static_want_gzip(struct wsgi_request *wsgi_req, char *filename, size_t *filename_len, struct stat *st) {
+    char can_gzip = 0, can_br =0;
```
Comment (xrmx) [inline core/static.c]: spaces instead of tabs

Comment (funkybob) [inline core/static.c]: Fixed

File: core/static.c [hunk 2]
```diff
@@ -42,17 +48,23 @@ int uwsgi_static_want_gzip(struct wsgi_request *wsgi_req, char *filename, size_t
...
+		*filename_len -= 3;
+		filename[*filename_len] = 0;
+	}
 
-	memcpy(filename + *filename_len, ".gz\0", 4);
-	*filename_len += 3;
-	
-	if (stat(filename, st)) {
+	if (can_gzip) {
+		memcpy(filename + *filename_len, ".br\0", 4);
```
Comment (xrmx) [inline core/static.c]: .gz no?

**Comment by xrmx** [comment]
@funkybob does this looks fine for you?https://github.com/xrmx/uwsgi/commit/440680acd6db659e42622f1e9e834bc0f9a2ef42

**Comment by xrmx** [comment]
Merged it by hand, closing the PR.

**Pull request closed without merging**